static/lang/re-en.json

```diff
@@ -3355,6 +3355,99 @@
                 "Tenth": "10th-Rank",
                 "Third": "3rd-Rank"
             },
+            "SpellTrickster": {
+                "AgileHand": {
+                    "CriticalFailureNote": "If you critically fail, the spell ends and you can't use this modification again for 24 hours.",
+                    "Description": "When you cast @UUID[Compendium.pf2e.spells-srd.Item.pwzdSlJgYqN7bs2w]{Telekinetic Hand}, you can modify its target to be a set of thieves' tools. When you @UUID[Compendium.pf2e.actionspf2e.Item.aBQ8ajvEBByv45yz]{Cast the Spell} in this way, the tools move up to 20 feet towards a device or lock. After you @UUID[Compendium.pf2e.actionspf2e.Item.3f5DMFu8fPiqHpRg]{Sustain} the spell on future turns, you can use the tools to attempt @Check[type:thievery] checks to @UUID[Compendium.pf2e.actionspf2e.Item.cYdz2grcOcRt4jk6]{Disable a Device} or @UUID[Compendium.pf2e.actionspf2e.Item.2EE4aF4SZpYf0R6H]{Pick a Lock} within the spell's range, but you take a -2 penalty to your Thievery check. If you critically fail, the spell ends and you can't use this modification again for 24 hours.",
```

Comment:
I'm not sure we need to link back to the spell the alteration will hit, since people can see the base text anyways.